services/validation.service.js

```diff
@@ -5,16 +5,20 @@ const {
   validateDeclaration,
   validatePostCode,
   validateEstablishmentFirstLine,
-  validateStreet
+  validateStreet,
+  validateTown,
+  validateEstablishmentTradingName
 } = require("./validationFunctions");
 
 const errorMessages = {
   declaration1: "You must tick all the declarations before continuing",
   declaration2: "You must tick all the declarations before continuing",
   declaration3: "You must tick all the declarations before continuing",
-  establishment_postcode: "Not a valid Postcode",
   establishment_first_line: "Not a valid First Line of address",
-  establishment_street: "Not a valid street name"
+  establishment_street: "Not a valid Street name",
```

Comment:
I'm not sure about this capitalisation - ask Flora whether we should be using sentence case